src/testing/docker/docker-compose-test.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+install/docker-compose up
```

Comment:
This can't work, there is no executable called `docker-compose` in the `install` folder.

You will have to run
```
cd install
docker-compose up -d
```
and then curl